src/Orchard.Web/Modules/Orchard.Projections/Providers/Builders/MemberBindingsStep.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Xml.Linq;
+using Orchard.Data;
+using Orchard.Localization;
+using Orchard.Projections.Services;
+using Orchard.Recipes.Services;
+
+namespace Orchard.Projections.Providers.Builders
+{
```

Comment:
Better use curly braces on the same line following style used in Orchard for consistency for all the files in this PR